main/monmon.rb

```diff
@@ -9,5 +10,9 @@
 
 puts('-----------------------------------------')
 balance.each do |key, val|
```

Comment:
Unsupported currencies should not go into `balance` structure, since it may introduce bugs later. For example, on the next stage we need to convert all currencies to the "main" currency. You can forget about filter of not supported currencies there and as the result it may calculate everything improperly or fail at all.

Reply:
I fix

Reply:
I don't see that you changed anything. The `balance` still holds both currencies with errors and without errors.